Scripts/DatabankLib/settings/molecules.py

```diff
@@ -0,0 +1,424 @@
+"""
+:module: settings/molecules.py
+:description: Module file with definition of different global-level dictionaries.
+
+There is a dictionary of lipids, ions, etc. If you add a lipid which is not yet
+in the databank, you have to add it here!
+"""
+
+# Dictionary of possible lipids
+lipids_dict = {
```

Comment:
I personally think that we should get rid of these dicts completely because they logicaly belong to data part and not to code part. That we started already to discuss with @ohsOllila. 

I mentioned it in the "restructuring" issue:
https://github.com/NMRLipids/Databank/issues/200#issuecomment-2545808276